src/lib.rs

```diff
@@ -145,6 +161,26 @@ impl Sdl2Window {
         Ok(window)
     }
 
+    /// Initialize the joystick subsystem. Required before joystick input
+    /// events will be returned. Returns the number available or error.
+    pub fn init_joysticks(&mut self) -> Result<u32, String> {
```

Comment:
yeah, this was just ported from the example in rust-sdl2. I don't use the return value in any of my stuff.